torch/csrc/jit/init.cpp

```diff
@@ -185,6 +185,16 @@ void initJITBindings(PyObject* module) {
                   param_name,
                   getQParamFunc,
                   at::ScalarType::QInt8);
+            } else if (param_name == std::string("bias")) {
+              auto getQParamFunc =
+                  py::cast<std::function<std::tuple<std::string, float, int>(
+                      float, float)>>(pyGetQParamFunc);
+              InsertQuantDequantNodesForParam(
+                  moduleObj,
+                  method_name,
+                  param_name,
+                  getQParamFunc,
+                  at::ScalarType::QInt32);
```

Comment:
The only difference between true and false branches is the last argument of `InsertQuantDequantNodesForParam` - could you please extract the common part out of the `if` statement?

Also, the function silently ignores other values of `param_name`. If someone has a typo in the name, the function would do nothing and not even complain about it - I suggest adding an assert.

Reply:
cast is also different. Done for assert

Reply:
The fact that the function signature is different for two cases seems really unfortunate since it causes a lot of code duplication. Any chance of refactoring it?

Reply:
So idea is that eventually as we expand to different quant schemes, we will have more cases here for weight and bias so more function signature is unavoidable. The interface will continue to stay same because it takes in template as function.

Else other approach was to have explicit pass for weight and bias but not using template would lead to explosion of torch apis in quantization.h
Lets discuss once u r back in detail how we can generalize this